dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/utils/otelCollector/OtelCollectorContainer.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal.utils.otelCollector;
+
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.containers.Network;
+
+/**
+ * A container for the OtelCollector trace server
+ * <p>
+ * Usage:
+ *
+ * <pre>
+ * {@code @ClassName}
+ * public static OtelCollectorContainer OtelCollectorContainer = new OtelCollectorContainer().withLogConsumer(new SimpleLogConsumer(MyTest.class, "OtelCollector"));
+ * </pre>
+ *
+ */
+public class OtelCollectorContainer extends GenericContainer<OtelCollectorContainer> {
+
+    public static final DockerImageName IMAGE_NAME = DockerImageName.parse("otel/opentelemetry-collector:0.74.0");
+
+    public static final int OTLP_GRPC_PORT = 4317;
+
+    private static final ClassLoader classLoader = OtelCollectorContainer.class.getClassLoader();
+
+    public OtelCollectorContainer() {
+        this(IMAGE_NAME);
+    }
+
+    public OtelCollectorContainer(DockerImageName imageName) {
+        super(imageName);
+        withExposedPorts(OTLP_GRPC_PORT);
+        withFileSystemBind("lib/LibertyFATTestFiles/otel-collector-config.yaml", "/etc/otel-collector-config.yaml", BindMode.READ_ONLY);
```

Comment:
I think this only works when using docker locally. Have you tested this with a remote docker (as will be used in the build)